typings/index.d.ts

```diff
@@ -1098,7 +1100,7 @@ declare module 'discord.js' {
     public createdTimestamp: number | null;
     public readonly expiresAt: Date | null;
     public readonly expiresTimestamp: number | null;
-    public guild: Guild | null;
+    public guild: InviteGuild | Guild | null;
```

Comment:
remove `| Guild`?

Reply:
can still be a regular guild, if resolved from cache

Reply:
Since `InviteGuild` now extends `BaseGuild`, this comment is now resolved.